2022-08-30 15:05:59.915 [http-nio-8001-exec-1] ERROR c.f.s.exceptionHandler.ProjectExceptionHandler - Could not get a resource from the pool; nested exception is io.lettuce.core.RedisConnectionException: Unable to connect to 192.168.40.128:6379
2022-08-30 15:06:09.927 [http-nio-8001-exec-4] ERROR c.f.s.exceptionHandler.ProjectExceptionHandler - Could not get a resource from the pool; nested exception is io.lettuce.core.RedisConnectionException: Unable to connect to 192.168.40.128:6379
2022-08-30 15:06:19.940 [http-nio-8001-exec-3] ERROR c.f.s.exceptionHandler.ProjectExceptionHandler - Could not get a resource from the pool; nested exception is io.lettuce.core.RedisConnectionException: Unable to connect to 192.168.40.128:6379
2022-08-30 15:06:29.950 [http-nio-8001-exec-2] ERROR c.f.s.exceptionHandler.ProjectExceptionHandler - Could not get a resource from the pool; nested exception is io.lettuce.core.RedisConnectionException: Unable to connect to 192.168.40.128:6379
2022-08-30 15:06:39.959 [http-nio-8001-exec-6] ERROR c.f.s.exceptionHandler.ProjectExceptionHandler - Could not get a resource from the pool; nested exception is io.lettuce.core.RedisConnectionException: Unable to connect to 192.168.40.128:6379
2022-08-30 15:06:49.968 [http-nio-8001-exec-5] ERROR c.f.s.exceptionHandler.ProjectExceptionHandler - Could not get a resource from the pool; nested exception is io.lettuce.core.RedisConnectionException: Unable to connect to 192.168.40.128:6379
